packages/requisitions/src/RequestRequisition/DetailView/Toolbar/ToolbarActions.tsx

```diff
@@ -1,49 +1,106 @@
 import React from 'react';
 import {
+  Switch,
```

Comment:
Maybe this would be better in the hoverclick popover thingy? Though, there's also an issue to add a filter on item name, so still need to add a search input on the left unless that's also hidden away